### ISSUE TITLE
select round(100,2) display 100 not is 100.00

### ISSUE THREAD

**Issue by MasonYangD** [ClickHouse/ClickHouse#33440]
This is an example from the official website of clickhouse.

![image](https://user-images.githubusercontent.com/38834622/148479164-4ec8ab3f-48be-49ac-b010-e9af9cb1db09.png)

This is mine.
![image](https://user-images.githubusercontent.com/38834622/148479231-60ea3b69-6348-4e0c-9fd3-a54bc854a88c.png)
Who can tell me , why?
I don't want a temporary setup, I want a permanent setup

_Originally posted by @MasonYangD in https://github.com/ClickHouse/ClickHouse/issues/33430#issuecomment-1007079231_

**Comment by MasonYangD** [comment]
https://github.com/ClickHouse/ClickHouse/issues/33430#issuecomment-1007077785

**Comment by den-crane** [comment]
```sql
set output_format_decimal_trailing_zeros = 1;
select toDecimal64(1, 6);
┌─toDecimal64(1, 6)─┐
│          1.000000 │
└───────────────────┘

set output_format_decimal_trailing_zeros = 0;
select toDecimal64(1, 6);
┌─toDecimal64(1, 6)─┐
│                 1 │
└───────────────────┘
```

ClickHouse release v21.9, 2021-09-09
Backward Incompatible Change

Do not output trailing zeros in text representation of Decimal types. Example: 1.23 will be printed instead of 1.230000 for decimal with scale 6. This closes #15794. It may introduce slight incompatibility if your applications somehow relied on the trailing zeros. Serialization in output formats can be controlled with the setting output_format_decimal_trailing_zeros. Implementation of toString and casting to String is changed unconditionally. #27680 (alexey-milovidov).
